include/net/mptcp.h

```diff
@@ -905,6 +905,8 @@ void mptcp_get_default_scheduler(char *name);
 int mptcp_set_default_scheduler(const char *name);
 bool mptcp_is_available(struct sock *sk, const struct sk_buff *skb,
 			bool zero_wnd_test);
+int mptcp_dont_reinject_skb(const struct tcp_sock *tp, const struct sk_buff *skb);
+bool mptcp_is_temp_unavailable(struct sock *sk, const struct sk_buff *skb, bool zero_wnd_test);
```

Comment:
This line needs to be removed as well if `mptcp_is_temp_unavailable` becomes `static` again.